### ISSUE TITLE
Systray icon?

### ISSUE THREAD

**Issue by colemickens** [BusyJay/gpg-bridge#4]
Hi @BusyJay!

Would you be open to have a mode where `gpg-bridge --systray` causes it to launch in the background, with a systray icon to show it's running and allow stopping?

I'm not sure I have the Rust+Windows skills for it, but I might try it if you're interested.

**Comment by BusyJay** [comment]
I thought about it, but couldn't think of what can do with a systray icon, though it was clearly more expressive than a silent background job.

That being said, I would be happy to merge it if you would like to contribute.

**Comment by colemickens** [comment]
Hm, I looked at the code and realized you already have a `--detach`. It _is_ kind of hard to justify a systray icon, other than a quick visual liveness check. I'm also worried about it bringing it a bunch of dependencies. I'll think about it more I guess.

**Comment by emmatyping** [comment]
I was thinking it would be nice to make this a windows service via https://github.com/mullvad/windows-service-rs so that it will e.g. start with Windows. Adding a tray icon to control the service also seems nice.

**Comment by BusyJay** [comment]
I agree tray icon is a good idea and now with https://github.com/microsoft/windows-rs, it should not be hard to implement.
